packages/design-system/src/components/Alert/Alert.tsx

```diff
@@ -172,69 +143,34 @@ export class Alert extends React.PureComponent<
     }
   }
 
-  render(): JSX.Element {
-    const {
-      children,
-      className,
-      autoFocus,
-      heading,
-      headingId,
-      headingLevel,
-      hideIcon,
-      alertRef,
-      role,
-      variation,
-      weight,
-      analytics,
-      analyticsLabelOverride,
-      ...alertProps
-    } = this.props;
-
-    const classes = classNames(
-      'ds-c-alert',
-      hideIcon && 'ds-c-alert--hide-icon',
-      variation && `ds-c-alert--${variation}`,
-      weight && `ds-c-alert--${weight}`,
-      className
-    );
-
-    const a11yLabel = (
-      <span className="ds-c-alert__a11y-label ds-u-visibility--screen-reader" id={this.a11yLabelId}>
-        {t(`alert.${variation ?? 'defaultLabel'}`)}:{' '}
-      </span>
-    );
-
-    return (
-      <div
-        className={classes}
-        ref={(ref) => {
-          this.alertTextRef = ref;
-          if (autoFocus) {
-            this.focusRef = ref;
-          } else if (alertRef) {
-            alertRef(ref);
-          }
-        }}
-        tabIndex={alertRef || autoFocus ? -1 : null}
-        role={role}
-        aria-labelledby={heading ? this.headingId : this.a11yLabelId}
-        {...alertProps}
-      >
-        {this.getIcon()}
-        <div className="ds-c-alert__body" id={this.headingId}>
-          {heading ? (
-            <div className="ds-c-alert__header ds-c-alert__heading">
-              {a11yLabel}
-              {this.heading()}
-            </div>
-          ) : (
-            a11yLabel
-          )}
-          {children}
-        </div>
+  return (
+    <div
+      className={classes}
+      ref={mergeRefs([alertRef, focusRef])}
+      tabIndex={alertRef || autoFocus ? -1 : null}
+      role={role}
+      aria-labelledby={heading ? headingId : a11yLabelId}
+      {...alertProps}
+    >
+      {getIcon()}
+      <div className="ds-c-alert__body" id={headingId} ref={bodyRef}>
+        {heading ? (
+          <div className="ds-c-alert__header ds-c-alert__heading">
```

Comment:
If we want to lean into analytics starting to show the a11y labels consistently, the `headingRef` should actually go on this element.